1.0-specification.norg

```diff
@@ -1184,8 +1184,8 @@ version: 1.0
    Aside from the details described above, free-form attached modifiers have the following properties:
    - Backslashes (`\`) are treated as verbatim and do not mean an {*** escaping}[escape sequence]
      (see {* precedence}).
-   - Despite being called "free-form" they can still only span a single paragraph (see {# attached
-     modifier range}).
+   - Despite being called "free-form" they can still only span a single paragraph (see {# free-form
+   attached modifiers})
```

Comment:
I believe it's mean to link to [this](https://github.com/nvim-neorg/norg-specs/commit/ebb525b653e7924e8f730d683400718332530bd6#diff-b42089c85b812e21373fc2ab8e63fc10306a2bbfffe72449494d585bb284ae35L841) although it still doesn't link correctly nor does it mention anything about paragraphs